core/Cargo.toml

```diff
@@ -79,6 +80,7 @@ version = "0.4.42"
 default = []
 lzma = ["lzma-rs", "swf/lzma"]
 avm_debug = []
+avm_profile = ["dep:tracy-client"]
```

Comment:
I'm proposing naming it like `tracy_avm`; it should be easier to understand/remember considering this always needs `tracy` features to be available.

...actually, wait, doesn't this feature need to be manually enabled in the toml? Can't we propage it from `desktop` crate, so we can do `--features=tracy,tracy_avm` when building?

Reply:
When building you can do `ruffle_core/foo` to enable the foo feature of Ruffle core

Reply:
Renamed to `tracy_avm`